### ISSUE TITLE
I think polyline individual nodes can't be deleted.

### ISSUE THREAD

**Issue by shujaatak** [hash1018/ShCAD#3]
I noticed that polyline individual nodes can't be deleted instead the whole object/polyline is deleted upon deletion. It would have been great if ShCAD had the ability to delete individual nodes an a polyline.

**Comment by hash1018** [comment]
Yes I know that, That is still on my ToDo list.  By the way Thank yor for your attention. ^^

**Comment by hash1018** [comment]
> I noticed that polyline individual nodes can't be deleted instead the whole object/polyline is deleted upon deletion. It would have been great if ShCAD had the ability to delete individual nodes an a polyline.

Hello, It's been a while. I just published pre-release for ShCAD.  some features are added.
but polyline function is removed for temporary.

I will recover polyline soon.

**Comment by shujaatak** [comment]
Waiting for the polyline function to be implemented.

You are doing great! Your programming skills are awesome!
can you please mention the tutorial/link/book/or any reference material to follow to learn Design Patterns in C++?

**Comment by hash1018** [comment]
> Waiting for the polyline function to be implemented.
> 
> You are doing great! Your programming skills are awesome!
> can you please mention the tutorial/link/book/or any reference material to follow to learn Design Patterns in C++?

https://g.co/kgs/sEyxd5  <- This is the one I study for design pattern. 
The second pre-release ShCAD is applied to more than 17 design patterns. 
It's good to improve skills by doing project.

**Comment by shujaatak** [comment]
> The second pre-release ShCAD is applied to more than 17 design patterns.

Outstanding! You are a champion!

**Comment by hash1018** [comment]
> > The second pre-release ShCAD is applied to more than 17 design patterns.
> 
> Outstanding! You are a champion!

Hello, its been a while again. I entered your repo, there is a QFFmpegWidget I am really looking for and  I want to make at this moment.

I am making project for screen recoder and I will add this with my shCAD. 
It's like when I render monitor screen or some other program such as power point, excel, or etc then add my ShCAD Graphic logic to them,  Sum all image then encode with ffmpeg, 

I am new to ffmpeg , now what i am doing is that I decode video file , get frame (raw data) but it's usaually yuv420 format which is can not be rendered with openGL directly , I tried to scale YUV420 to rgba format with sws_scale but this is literally software scaler which is quite takes so long. 

I need to convert Yuv format to any format OpenGL can render as fast as video file's frame Per second.   

I found one solution is that Shader in openGL to convert YUV420 to rgba or someting thats uses GPU pipeline.  am I doing this right?

**Comment by hash1018** [comment]
> > The second pre-release ShCAD is applied to more than 17 design patterns.
> 
> Outstanding! You are a champion!

Hello, Let me introduce you my second project Adora which is video-recorder based on qt, ffmpeg, and C++.

it's working on Windows only at the moment.

**Comment by shujaatak** [comment]
> Hello, Let me introduce you my second project Adora which is video-recorder based on qt, ffmpeg, and C++.

Awesome! Going to check it out!

**Comment by shujaatak** [comment]
> it's working on Windows only at the moment.

Right now I am using Linux, I will check Adora when I turn on my Windows PC.

**Comment by hash1018** [comment]
> > it's working on Windows only at the moment.
> 
> Right now I am using Linux, I will check Adora when I turn on my Windows PC.

after using let me know how you feel and give me any advice
